src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -186,7 +186,7 @@ public CompilerOptions getCompilerOptions() {
     // Late Provides are errors by default, but they do not prevent clutz from transpiling.
     options.setWarningLevel(DiagnosticGroups.LATE_PROVIDE, CheckLevel.OFF);
 
-    options.setLanguage(LanguageMode.ECMASCRIPT_2017);
+    options.setLanguage(LanguageMode.ECMASCRIPT_NEXT);
```

Comment:
Is ES_NEXT the "floating latest", or just an alias for some version? Maybe add a comment explaining that?

Reply:
It's the floating latest. Added a comment.